haiku-apps/copynametoclipboard/copynametoclipboard-1.0.1.recipe

```diff
@@ -9,7 +9,8 @@ COPYRIGHT="2000 Werner Freytag
 	2012 Humdinger"
 LICENSE="MIT"
 REVISION="2"
-SOURCE_URI="$(HOMEPAGE)/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
+SOURCE_URI="https://github.com/HaikuArchives/CopyNameToClipboard/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
+SOURCE_DIR="CopyNameToClipboard-1da1d52429db53f15c3debf817e982d9ce6346b4"
```

Comment:
The terminal output didn't provide me with a CHECKSUM_SHA256 to put in the recipe, is there another way I can get it?